src/Admin/CollectionAdmin.php

```diff
@@ -62,7 +62,7 @@ protected function configureFormFields(FormMapper $formMapper)
             ])
         ;
 
-        if (interface_exists('Sonata\MediaBundle\Model\MediaInterface')) {
+        if (interface_exists(MediaInterface::class)) {
```

Comment:
Import is missing

Reply:
I always check with code inspection for undefined classes, but yea - missed this one, thanks!